src/MongoDB.Driver.Core/Core/Servers/ServerDescription.cs

```diff
@@ -459,6 +459,7 @@ public override string ToString()
                 .AppendFormatIf(_state == ServerState.Connected, ", WireVersionRange: \"{0}\"", _wireVersionRange)
                 .AppendFormatIf(_electionId != null, ", ElectionId: \"{0}\"", _electionId)
                 .AppendFormatIf(_heartbeatException != null, ", HeartbeatException: \"{0}\"", _heartbeatException)
+                .AppendFormat(", LastUpdateTimestamp: \"{0}\"", _lastUpdateTimestamp)
```

Comment:
What format is _lastUpdateTimestamp in? Is it culture neutral?

I would recommend the following format to avoid any ambiguity:

2019-03-26T22:56:12.345Z

Not sure how many decimals we should use.